src/sqlfluff/dialects/dialect_snowflake_keywords.py

```diff
@@ -524,4 +527,9 @@
 WRITE
 XML
 ZONE
-ZSTD"""
+ZSTD
+"""
+
+# These are not official Snowflake keywords, but SQLFluff needs them to parse well.
+snowflake_unreserved_keywords += """IDENTIFIER
+"""
```

Comment:
We often add unreserved words (e.g. `ABORT`) so no need to pull them into a separate section. Too much to track that way. So can you move this above?

Reply:
No problem.